ios/Podfile

```diff
@@ -0,0 +1,90 @@
+# Uncomment this line to define a global platform for your project
```

Comment:
This file is generated while I opened the project in VSCode on my mac. I guess it wasn't there initially because @chansen424 does not run it on mac.